src/errors.js

```diff
@@ -1,6 +1,7 @@
-export function RuntimeError(message, input) {
+export function RuntimeError(message, input, cause = undefined) {
```

Comment:
```suggestion
export function RuntimeError(message, input, cause) {
```

Reply:
Durr, yes, vanilla JS.

Reply:
It’s just that the default value is already undefined, so setting the default to undefined has no effect.

Reply:
Yeah, my brain was in TS land.